src/pages/bridge/TransferDialog.tsx

```diff
@@ -69,66 +76,72 @@ function BaseTransferDialog(props: BaseTransferDialogProps) {
   }, [transfer]);
 
   return (
-    <Modal id={id} open={visible} className="bg-base-200">
+    <Modal id={id} open={visible} className="transfer-dialog rounded-md px-2">
       <CloseButton onClick={onClose} />
       <Modal.Body>
         <div className="flex flex-col items-center justify-between">
           {statusIcon}
-          <div className="mt-4" />
-          <h1 className="text-xl mb-5">{title}</h1>
+          <div className="mt-5" />
+          <h1 className="text-2xl transfer-dialog-contrast-text font-semibold mb-1">{title}</h1>
           {content}
-          <div id="details" className="rounded flex flex-col p-5 mt-1 w-11/12">
-            <div className="flex flex-row justify-between">
+          <Divider className="mx-5 mb-2 mt-1" />
+          <div
+            id="details"
+            tabIndex={0}
+            onClick={toggle}
+            className={`collapse collapse-arrow rounded-lg bg-black bg-opacity-3 transfer-dialog-text flex flex-col w-11/12 ${collapseVisibility}`}
+          >
+            <div className="collapse-title flex flex-row justify-between">
               <div className="text-sm">Bridge fee</div>
               <div className="text-sm">{nativeToDecimal(transfer.original.fee.toNumber()).toString()}</div>
             </div>
-            <div className="flex flex-row justify-between">
-              <div className="text-sm">Destination Address (Stellar)</div>
-              <CopyableAddress
-                inline={true}
-                className="text-sm p0"
-                variant="short"
-                publicKey={destinationStellarAddress}
-              />
-            </div>
-            <div className="flex flex-row justify-between">
-              <div className="text-sm">Vault Address ({tenantNameCapitalized})</div>
-              <CopyableAddress
-                inline={true}
-                className="text-sm p-0"
-                variant="short"
-                publicKey={transfer.original.vault.accountId.toString()}
-              />
-            </div>
-            {vaultStellarPublicKey && (
+            <div className="collapse-content space-y-4">
               <div className="flex flex-row justify-between">
-                <div className="text-sm">Vault Address (Stellar)</div>
+                <div className="text-sm">Destination Address (Stellar)</div>
                 <CopyableAddress
                   inline={true}
-                  className="text-sm p-0"
+                  className="text-sm p0"
                   variant="short"
-                  publicKey={vaultStellarPublicKey}
+                  publicKey={destinationStellarAddress}
                 />
               </div>
-            )}
-            {showMemo && (
               <div className="flex flex-row justify-between">
-                <div className="text-sm">Memo</div>
+                <div className="text-sm">Vault PEN Address ({tenantNameCapitalized})</div>
```

Comment:
We should not call it PEN address. I would actually suggest we completely remove that and only call it `Vault Address ({tenant})`.